staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook_v1_test.go

```diff
@@ -40,6 +40,7 @@ import (
 	authorizationv1 "k8s.io/api/authorization/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
+	authzconfig "k8s.io/apiserver/pkg/apis/apiserver"
```

Comment:
The most consistent approach we have is to use the last N path segments for import alias. In this case, I would just remove import alias and use `apiserver.`